src/sentry/static/sentry/app/components/avatar/suggestedAvatarStack.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {css} from '@emotion/core';
+import styled from '@emotion/styled';
+
+import ActorAvatar from 'app/components/avatar/actorAvatar';
+import BaseAvatar from 'app/components/avatar/baseAvatar';
+import {Actor} from 'app/types';
+
+type Props = {
+  owners: Actor[];
+};
+
+// Constrain the number of visible suggestions
+const MAX_SUGGESTIONS = 5;
+
+const SuggestedAvatarStack = ({owners, ...props}: Props) => {
```

Comment:
Can we be more explicit with `Props` here irt the `props` spreading? I think it should accept props of ActorAvatar and BaseAvatar as well?